src/APIClient.php

```diff
@@ -70,6 +70,16 @@ public static function getTypeName($nb = 0)
         return _n("API client", "API clients", $nb);
     }
 
+    public static function getSectorizedDetails(): array
+    {
+        return ["config", "config", __CLASS__];
```

Comment:
We used to use the `self::class` instead of `__CLASS__` now.

```suggestion
        return ["config", Config::class, self::class];
```